datajunction-server/tests/api/sql_test.py

```diff
@@ -828,7 +830,7 @@ def _validate_access(access_control: access.AccessControl):
         return _validate_access
 
     app = client_with_roads.app
-    app.dependency_overrides[validate_access] = validate_access_override
+    app.dependency_overrides[validate_access_placeholder] = validate_access_override
```

Comment:
Hmm, can we continue to call this dependency `validate_access`, both so that there are fewer breaking changes downstream and so that when we do implement this, we don't need to rename it from "placeholder"?

Reply:
Oh thanks for catching this. I originally did this because I named the other function `validate_access` but I eventually changed that to `validate_access_requests` to be more explicitly named so I'll change this back.

Reply:
Fixed 👍